### PR TITLE
Remove misleading statement

### PR DESCRIPTION
**Pull request by darobin**
The cited blog post according to which publishers would be harmed at n% by the removal of cookies is plainly and simply bad science. The way in which one can evaluate the value of cookie tracking is by comparing a world in which it is available to a world in which it isn't. What the linked blog does is measure the value of cookies in a world in which people can bid on them elsewhere. The findings it trumpets is not surprising to anyone, it is also completely unrelated to the statement being made. It tells us nothing about what the impact on publishers would be if Chrome were to implement ITP. On the contrary, there are many good reasons to expect that it would in fact be beneficial.

### CODE REVIEW

**Comment by michaelkleber** [comment]
Robin,

As [TechCrunch](https://techcrunch.com/2019/08/22/google-proposes-new-privacy-and-anti-fingerprinting-controls-for-the-web/) and [Digiday](https://digiday.com/media/the-scale-of-the-problem-is-enormous-apple-flexes-strong-anti-tracking-stance/) both pointed out in their coverage today, the "52 percent less revenue" number in that blog post is well in line with what real-world publishers have seen on their Safari traffic due to ITP.

Of course you're right that the equilibrium will adjust if the market changes, and maybe (per Digiday's sources' speculation) the real answer would only be a 40%, not 52%, rev loss.  It's hard to know.

Personally, I would be delighted if the number were smaller!  As the guy whose job it is to figure out privacy-preserving ways to narrow this gap:  The smaller that number is, the easier my job becomes.

But until we have some better science, I need to evaluate the world with the data I have, not the data I'd like to have.

**Comment by darobin** [comment]
I'm sorry but nothing you are saying has any bearing on the issue. The data you have is simply unrelated to the statement you are making. I sympathise that you'd like to have better data, but you can't just grab something randomly down the aisle and pretend it'll do.

Let's assume you're facing an obesity epidemic that's killing kids and you are thinking of controlling what food can be sold around schools. You've made an A/B test by having half the restaurants around schools sell balanced food they don't know how to cook and since (surprise!) kids are flocking to the junk you are reaching the conclusion that, whoa, if kids were to eat a balanced diet the food industry would go out of business!

This is not about equilibria, this is simply not an experiment that supports either your thesis or its opposite. It is, in fact, not even wrong.

**Comment by michaelkleber** [comment]
It sounds like you're asserting that "what happened to publisher revenue when Safari launched ITP" is not a useful predictor of "what would happen to publisher revenue when [any other browser] launches ITP".  Can you explain why?  Something with more care and substance than "you're not even wrong" would be great.

If I'm following your "kids flocking to junk food" analogy, then it does not seem on point to me.  It might make sense in a world where kids don't care about how many restaurants they can eat at — er, sorry, this analogy is too strained, let me try again.  It might make sense in a world where advertisers don't care what fraction of people actually see their ads.  But advertisers _do_ care an extraordinary amount about the reach of their ad campaigns.  A prediction that says they would be content simply not selling stuff to iPhone users is unrealistic.

**Comment by colinclerk** [comment]
@michaelkleber "what happened to publisher revenue when Safari launched ITP" is not necessarily a useful predictor in a world where advertisers can target based on browser.  It might be that advertisers are targeting non-Safari, and once all browsers implemented ITP revenue would even back out.

Relevant questions:
- What portion of Google's "randomly selected fraction of traffic" was Safari traffic?  
- Is there any evidence that advertising dollars are shifting to other mediums?  
- Has the cost of advertising increased in non-Safari browsers?

**Comment by michaelkleber** [comment]
@colinclerk The hypothesis that advertisers are content to just give up on Safari users doesn't seem particularly believable based on my experience.  But if you have data to share, please do so!

**Comment by colinclerk** [comment]
@michaelkleber isn't "giving up on Safari users" effectively what drove that 52% number?

It appears Google allows advertisers to define a "remarketing" audience and only target that cohort:
https://support.google.com/google-ads/answer/2453998?hl=en

Since remarketing is impossible when cookies are disabled, I would assume that this feature is being highly leveraged and was the primary driver of that 52% revenue drop.

What percent of advertising dollars gets spent on remarketing vs non-remarketing?  If the vast majority is on remarketing, then for now it would seem like most advertisers are giving up on Safari users.  Though I would expect that to even out if remarketing were disabled at large.

Ad networks should have the data to prove/disprove my assumption.  Can you share anything more than the 52% stat?

**Comment by michaelkleber** [comment]
There are multiple types of ad targeting that make use of user identity today.  The [FLoC explainer](https://github.com/jkarlin/floc), linked from the "small amounts of cross-site information" part of this doc, is an example of how to make one such targeting type (IBA, "Interest-Based Advertising") possible even in a world where you can't recognize the same user on multiple sites.

So let's over-simplify.  Ad prices are the result of an auction, and more participants in the auction leads to a higher winning price.  When someone who's recognized as a motorcycle enthusiast reads a news article about the Amazon rainforest fires, the motorcycle-related ads have to fight it out with the news-related ads, so the winner often has to pay more than when a generic person (with no known interests) reads about the fires.

In a world where the only way to show ads to motorcycle enthusiasts is while they are visiting motorcycle-related sites, the news sites just get less money.  It doesn't matter if the motorcycle advertisers can recognize which of the news-site visitors "really" have no interests and which ones "just look like" they have no interests because of ITP.  They still can't effectively spend any of their dollars supporting the news sites, because they have no idea which news-site visitors care about motorcycle ads too.

Of course there are other dynamics in play too, but this is where most of where the 52% went: many fewer ads in the auction, many fewer sources of funding for web sites.

**Comment by colinclerk** [comment]
@michaelkleber Yes, I agree that there were fewer ads in the auction.  My assertion (and how I understood @darobin's analogy) is that there may be fewer ads in the auction primarily because advertisers can opt out and leverage a remarketed ad instead.

This "flocking" to remarketed ads is only an option as long as remarketing still exists.  If we get rid of remarketing altogether, I have a hard time believing that advertisers would simply stop publishing on news sites.

However, Google suggests that its measurement of a 62% drop in news-site publishing revenue _will hold true_ even if remarketing is removed altogether.  Do you agree with that figure, regardless of this contributing factor?

**Comment by michaelkleber** [comment]
As I said in my top comment, "Of course you're right that the equilibrium will adjust if the market changes, and maybe (per Digiday's sources' speculation) the real answer would only be a 40%, not 52%, rev loss. It's hard to know."  My point is not the specific number, it's that the A/B experiment gives us good reason to expect that the revenue loss would be very large for many sites.

> This "flocking" to remarketed ads is only an option as long as remarketing still exists. If we get rid of remarketing altogether, I have a hard time believing that advertisers would simply stop publishing on news sites.

Suppose that 10% of people who visit the news site care about motorcycles.  Are you expecting motorcycle advertisers to spend as much supporting the news site as they did before?  They would only expect that to generate 10% as much business, in your new world.  How could they afford to spend the same number of ad dollars for only 10% as many sales?

**Comment by colinclerk** [comment]
@michaelkleber It seems you agree with our methodology concerns, but have still concluded that the impact on publishers will be "very large."  Can you share how you're reaching that conclusion?

> Suppose that 10% of people who visit the news site care about motorcycles. Are you expecting motorcycle advertisers to spend as much supporting the news site as they did before? They would only expect that to generate 10% as much business, in your new world. How could they afford to spend the same number of ad dollars for only 10% as many sales?

The news site or ad network could learn which users read motorcycle-related content written by the news site, and allow the motorcycle advertiser to only target those users. Whether the motorcycle advertiser spends more or less with a given publisher will likely depend how much the news site writes about motorcycles.

Even if a motorcycle advertiser leaves, I would expect another advertiser to fill in the gap.  That expectation is informed by:

The NY times saying the lack of global identity tracking in EU hasn't impacted their revenue:
https://digiday.com/media/gumgumtest-new-york-times-gdpr-cut-off-ad-exchanges-europe-ad-revenue/

A study saying cookies only increase revenue by 4%:
https://weis2019.econinfosec.org/wp-content/uploads/sites/6/2019/05/WEIS_2019_paper_38.pdf

(Credit to https://freedom-to-tinker.com/2019/08/23/deconstructing-googles-excuses-on-tracking-protection/ for pointing me toward this data.)
